### PR TITLE
added file htmlassign 2Kaveri@work

### PR DESCRIPTION
**Pull request by kaverigond19945**

### CODE REVIEW

**Review by srigowrideo — APPROVED** [review]

File: kaverigond/HTML/html assign/kaveri_assign/Q2.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            <h3>
+             Notepad or TextEdit. 
+
+            </h3>
+            <p> 
+                Web pages can be created and modified by using professional HTML editors.
+                However, for learning HTML we recommend a simple text editor like Notepad (PC) or TextEdit (Mac).    
+            </p>
+
+        </body>
```
Comment (Pawan1005) [inline kaverigond/HTML/html assign/kaveri_assign/Q2.html:23]: html  tag closing missing

File: kaverigond/HTML/html assign/kaveri_assign/Q4.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+            <h1> Introduction </h1>
+            <p>
+            <h3>My name is Kaveri Gond</h3>
+            <h3>I am from Khamgaon</h3>
+            <h3>completed my BE</h3>
+            <h2> Thank You</h2>
+            </p>
+            
+        </body>
+    </head>
```
Comment (Pawan1005) [inline kaverigond/HTML/html assign/kaveri_assign/Q4.html:15]: wrong code 
head tag need to close before body tag
html closing tag is also missing

File: kaverigond/html assign 2/my 2ndhtml.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<DOCTYP html>
+ <html  style="background-color:DodgerBlue;">
+      <head>
+       <h1 style="text-align:center;"> Google </h1> 
+     </head>
+
+     <body><i> This article is about the company. </i></body>
+
+    <h2>From the garage to the Googleplex</h2> 
+    <body>The Google story begins in 1995 at <b> Stanford University.</b>
```
Comment (Pawan1005) [inline kaverigond/html assign 2/my 2ndhtml.html:10]: wrong body tag

File: kaverigond/html assign 2/my 2ndhtml.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<DOCTYP html>
+ <html  style="background-color:DodgerBlue;">
+      <head>
+       <h1 style="text-align:center;"> Google </h1> 
+     </head>
+
+     <body><i> This article is about the company. </i></body>
```
Comment (Pawan1005) [inline kaverigond/html assign 2/my 2ndhtml.html:7]: body tag needs to close at end of file before closing of html tag

File: kaverigond/html assign 2/my 2ndhtml.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        They called this search engine <i> Backrub. </i> 
+
+        <h2><b style="font-size:20px;"> Initial public offering </b></h2>
+
+        
+            Google's initial public offering (IPO) took place five years later, on 19 <sup>th</sup>  August 2004. At 
+            that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work together at Google for 
+            <del>30</del>  <u>20</u>  years, until the year 2024.
+            <p style="color: rgb(226, 43, 58);"> Thank You.</p> 
+        </body>
```
Comment (Pawan1005) [inline kaverigond/html assign 2/my 2ndhtml.html:29]: remove this body tag you can have only one body tag in code

File: kaverigond/html assign 2/my 2ndhtml.html [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        
+     </body>
+    </html>
```
Comment (Pawan1005) [inline kaverigond/html assign 2/my 2ndhtml.html:32]: add new line at the end of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by anitawankhade1994 — APPROVED** [review]

**Review by srigowrideo — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Review by srigowrideo — APPROVED** [review]

**Pull request merged**